CUSTOM_PROFILE/CUSTOM_PROFILE.info.yml.example

```diff
@@ -0,0 +1,86 @@
+# NOTE: Rename this file to replace CUSTOM_PROFILE with your profile name
+# and remove .example from the file name.
+name: Custom Profile Name
+type: profile
+description: 'Description of your custom profile.'
+core_version_requirement: '^9'
+
+# Optional: Declare your installation profile as a distribution
+# This will make the installer auto-select this installation profile.
+# The distribution_name property is used in the installer and other places as
+# a label for the software being installed.
+distribution:
+  name: Distribution Name
+
+# Your custom list of modules. You may remove Acquia CMS component modules and
+# add or remove other modules here as necessary for your custom profile.
+install:
+  - acquia_cms_common
+  - acquia_cms_search
+  - acquia_cms_article
+  - acquia_cms_image
+  - acquia_cms_document
+  - acquia_cms_person
+  - acquia_cms_place
+  - acquia_cms_toolbar
+  - acquia_cms_tour
```

Comment:
@kdruckman Please remove this, as this module is not public yet.

Reply:
ah yes! Thank you!